wolfcrypt/src/dsa.c

```diff
@@ -97,68 +97,133 @@ void wc_FreeDsaKey(DsaKey* key)
     mp_clear(&key->p);
 }
 
+
+/* validate that (L,N) match allowed sizes from FIPS 186-4, Section 4.2.
+ * l - represents L, the size of p in bits
```

Comment:
lowercase l on it's own is a poor variable name, looks like I and 1 depending on font.